### ISSUE TITLE
Allow empty lines between code blocks

### ISSUE THREAD

**Issue by hacknug** [gridsome/eslint-plugin-gridsome#43]
**Please describe what the rule should do:**

Right now adding an empty line between a `fragment` and a `query` results in the plugin warning you about it. I don't have a strong opinion about this but I feel like there might be no need to flag it as long as it's just _one_ empty line between code blocks.

Do you have any examples of `graphql` code where they enforce the no-empty-lines style or otherwise? What do you think?

I don't have any app with lots of fragments and queries so it's not a real issue for me but I feel it could be for other users or become one once core adds [support for more graphql-related stuff](https://github.com/gridsome/gridsome/issues/232).

**What category should the rule belong to?**

- [x] Enforces code style
- [ ] Warns about a potential error
- [ ] Suggests an alternate way of doing something
- [ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

This is what the correct code looks like as of v1.4.0:

```vue
<page-query>
  fragment RankingParts on TourRanking {
    id
    rankings: data {
      rank
      nickname
      score
      badge: pin_badge_image_url
      ranking_started_at
      ranking_finished_at
    }
  }
  query($id: ID, $prevId: ID, $nextId: ID) {
    ranking: tourRanking(id: $id) {
      ...RankingParts
    }
    next: tourRanking(id: $prevId) {
      ...RankingParts
    }
    prev: tourRanking(id: $nextId) {
      ...RankingParts
    }
  }
</page-query>
```

This is what it could look like:

```vue
<page-query>
  fragment RankingParts on TourRanking {
    id
    rankings: data {
      rank
      nickname
      score
      badge: pin_badge_image_url
      ranking_started_at
      ranking_finished_at
    }
  }

  query($id: ID, $prevId: ID, $nextId: ID) {
    ranking: tourRanking(id: $id) {
      ...RankingParts
    }
    next: tourRanking(id: $prevId) {
      ...RankingParts
    }
    prev: tourRanking(id: $nextId) {
      ...RankingParts
    }
  }
</page-query>
```

**Comment by tyankatsu0105** [comment]
@hacknug 
Thanks for the issue! And sorry for the late reply.

Umm.. This is a very interesting question.

`format-query-block` rule use Prettier's graphql parser.
I was tried Prettier playground. And I understood what you said.
[It's correct what you said!!](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAzATgQwOYFsEwAEASplANYCWU2ACpujAM6HSEAqEAruqRddkLAAOlELjClACaiJhLPxpMkhKZhiYhsufLLltcqJTDkomfAYlNI6OJfEAjTFOxwVAB2oB9Jy7hfKXBx-HgAbe11FbC8mDUY4KS91CIUqGi9UakomAAsEpJhLAF9tEqhRCrFCAEcuOHQATwAKABJpFQBJABEAGkIW91sANw6pTt7+qDgADxhR8YBKLSrxVIEVGG5ePQEm9v7pJZEViQA6c740ugZmYssp2Y2ty939geHRo4jz05eaekYTDuJ0GcCGTx4f2wezGkxmcykXxO4h+UIBtxOZXERRAPRAEHcMEo0CYyFADHQEAA7gCEKSUJghhBpLiQA4sCY4DAAMruTBgATIGDoOp4nIwXChADqOUo8CYfLAcG5dLllCGcoayHATFJeOoTHqMFoWDwmGQqEwoUNeIAVkxpgAhDnkLnc8xwAAy1DgFqtNpA9um3IEoTgAEUuBB4H7rXA8Xz0Ib0NrsFh3DlqqFWYNqDApdIYDlkAAOAAMCcphql6e1oOTQ19eNq0bgJoJ9JAmCYAFopgkEqzbLVKLYTTggrGA4bcJQhSL4yAmKGI1GY0hLXG8RoHAWpEXkAAmbdYSihAQAYQguEnKCg0CbIC4hvYmAc9M3AaGdQ6UCkBG5MB0EoQkAEE-25GAGjDKc4CKIogA)

I try refactor the rule.

But if you can't wait and you can refactor, please send PR👍

**Comment by tyankatsu0105** [comment]
@hacknug 
eslint-plugin-gridsome supports your suggestion's feature at last!
Yeah!

**Comment by hacknug** [comment]
Yaaaaaaaas 🎉  Thank you @tyankatsu0105 ❤️
